src/guides/v2.3/ui_comp_guide/concepts/magento-bindings.md

```diff
@@ -41,6 +42,9 @@ or
 
 If you use the Magento syntax, the Magento wrapper replaces the Magento syntax with the matching Knockout comments or `data-bind` attributes during the loading of the HTML template.
 
+{:.bs-callout-info}
+For render performance we recommended use knockout native syntax. Magento syntax will render slow than native knockout syntax. When use magento syntax every time HTML template is loaded for a component, it gets parsed and adjusted accordingly so that KnockoutJS actually understands which bindings it should apply. Both approaches use native syntax or magento syntax will work fine but in some case if performance is matter so native syntax is the best option
```

Comment:
```suggestion
For render performance, we recommend using knockout native syntax. Magento syntax will render slower than native knockout syntax. When using Magento syntax, every time the HTML template is loaded for a component, it gets parsed and adjusted accordingly so that KnockoutJS actually understands which bindings it should apply. Both approaches using native syntax or Magento syntax will work fine but in some cases if performance will matter so native syntax is the advised option.
```